bitnami/kibana/templates/deployment.yaml

```diff
@@ -97,6 +97,34 @@ spec:
             subPath: {{ .Values.persistence.subPath }}
             {{- end }}
         {{- end }}
+        - name: copy-default-plugins
+          image: {{ include "kibana.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.resources }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+          {{- else if ne .Values.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.resourcesPreset) | nindent 12 }}
+          {{- end }}
+          command:
+            - /bin/bash
+          args:
+            - -ec
+            - |
+                #!/bin/bash
+
+                . /opt/bitnami/scripts/libfs.sh
+                . /opt/bitnami/scripts/kibana-env.sh
+
+                if ! is_dir_empty "$SERVER_DEFAULT_PLUGINS_DIR"; then
```

Comment:
Where is `SERVER_DEFAULT_PLUGINS_DIR` expected to be defined? `kibana-env.sh` makes no reference to it. I _can_ see `SERVER_PLUGINS_DIR`, however.

This is causing the script to fail for us, although I can't guarantee we've configured everything correctly:

```
/opt/bitnami/scripts/libfs.sh: line 61: 1: missing directory
```

Reply:
It is defined here: https://github.com/bitnami/containers/blob/main/bitnami/kibana/8/debian-12/rootfs/opt/bitnami/scripts/kibana-env.sh#L96

Are you changing the default image?

Reply:
Thanks for clarifying. Indeed, we're overriding the tag:

```
    image:
      tag: 8.8.2
```

Seems that isn't a good idea, so we'll look to stop doing that. Thanks again :+1: